packages/frontend-asset-loader/src/__tests__/utils.test.ts

```diff
@@ -0,0 +1,129 @@
+import { logger } from "../utils/logger";
+import { isValidHashName, getDuplicateHashedFileName } from "../utils/utils";
+
+describe("isValidHashName", () => {
```

Comment:
No issues here, just wanted to say these test cases look great, really through 🙌